packages/contracts/src/networks.ts

```diff
@@ -6,11 +6,13 @@ export type SupportedNetwork = {
     apiURL: string
     browserURL: string
     envKey: string
+    supported: boolean
   }
   currency: string
   dripSize: string
   requiredEnvVariables: Array<string>
   networkType: NetworkType
+  dripVersion: number
```

Comment:
Added a `dripVersion` field which makes it easy for us to update the drip sizes on different networks. We just have to update the `dripSize`, bump the `dripVersion`, and then we can run the deploy script and it will archive any previous Drippie configurations and replace them with the new version